README.md

```diff
@@ -251,7 +251,7 @@ Response body
 
 #### Retrieve and store CA certificate
 ```shell
-curl --request GET 'http://<CAPIF_HOSTNAME>/ca-root' 2>/dev/null | jq -r '.certificate' -j > ca.crt
+curl --request GET 'http://<CAPIF_HOSTNAME>:8080/ca-root' 2>/dev/null | jq -r '.certificate' -j > <CA Certificate file>
```

Comment:
not always will be port 8080, for example, at deployment on openshift the port is 80, this is why in previous version of curl command I didn't setup a port at request to http endpoint. Maybe we can put something of <CAPIF_HTTP_PORT>=:8080